gui-libs/wlroots/wlroots-0.17.2.ebuild

```
@@ -0,0 +1,102 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Pluggable, composable, unopinionated modules for building a Wayland compositor"
+HOMEPAGE="https://gitlab.freedesktop.org/wlroots/wlroots"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://gitlab.freedesktop.org/${PN}/${PN}.git"
+	inherit git-r3
+	SLOT="0/9999"
+else
+	SRC_URI="https://gitlab.freedesktop.org/${PN}/${PN}/-/releases/${PV}/downloads/${P}.tar.gz"
+	KEYWORDS="amd64 arm64 ~loong ppc64 ~riscv x86"
```

please don't forget to unstable on bump

```suggestion
	KEYWORDS="~amd64 ~arm64 ~loong ~ppc64 ~riscv ~x86"
```

will fix on push